cpp/src/interop/to_arrow.cu

```diff
@@ -403,14 +436,37 @@ std::shared_ptr<arrow::Table> to_arrow(table_view input,
 
   return result;
 }
+
+std::shared_ptr<arrow::Scalar> to_arrow(cudf::scalar const& input,
+                                        column_metadata const& metadata,
+                                        rmm::cuda_stream_view stream,
+                                        arrow::MemoryPool* ar_mr)
+{
+  auto column = cudf::make_column_from_scalar(input, 1, stream);
+  cudf::table_view tv{{column->view()}};
+  auto arrow_table  = cudf::to_arrow(tv, {metadata}, stream);
+  auto ac           = arrow_table->column(0);
+  auto maybe_scalar = ac->GetScalar(0);
```

Comment:
I think most of this can be `const`?
```suggestion
  auto const column = cudf::make_column_from_scalar(input, 1, stream);
  auto const tv = cudf::table_view{{column->view()}};
  auto const arrow_table = cudf::to_arrow(tv, {metadata}, stream);
  auto const ac = arrow_table->column(0);
  auto const maybe_scalar = ac->GetScalar(0);
```

Reply:
Yup they can all be. Fixing.